lib/api/holdings_api.rb

```diff
@@ -15,6 +15,16 @@ class HoldingsAPI < Sinatra::Base
     "pong"
   end
 
+  post "/v1/reload_cached" do
```

Comment:
The issue here is that because there are many workers and many threads for the holdings API, calling this endpoint will reload the cache in _one_ of them, but you can't be guaranteed to reload it for _all_. I'm a bit torn on exactly how best to do it.